src/usr/local/www/interfaces.php

```diff
@@ -274,7 +274,10 @@ function remove_bad_chars($string) {
 		$pconfig['type6'] = "slaac";
 		break;
 	case "dhcp6":
-		$pconfig['dhcp6-duid'] = $wancfg['dhcp6-duid'];
+		if(empty($config['system']['global-v6duid'])) {
+			unset($wancfg['dhcp6-duid']);
+		}
```

Comment:
Could you explain further what you mean  by global option? What I am trying to prevent is the user selecting this option without entering a valid DUID; the wrong or no DUID may, in the case of a RAM disk cause dhcp6c to write a new DUID, this is what I'm trying to avoid as much as anything.